Sources/Interaction/Widgets/PiecewiseGaussianWidget/ComputeHistogram.worker.js

```diff
@@ -0,0 +1,17 @@
+import registerWebworker from 'webworker-promise/lib/register';
+
+registerWebworker(async ({ array, min, max, numberOfBins }, emit) => {
+  const delta = max - min;
+  const histogram = new Float32Array(numberOfBins);
```

Comment:
Any reason for float32 instead of uint32?

Reply:
Later, we normalize.